models/gif_cond_exp_multisynapse.h

```diff
@@ -0,0 +1,467 @@
+  [2]) the values of eta_i and gamma_i are affected immediately after spike
+  emission. However, GIF toolbox (http://wiki.epfl.ch/giftoolbox) which fits
+  the model using experimental data, requires a different set of eta_i and
+  gamma_i. It applies the jump of eta_i and gamma_i after the refractory period.
+  One can easily convert between q_eta/gamma of these two approaches:
+  q_eta_giftoolbox = q_eta_NEST * (1 - exp( -tau_ref / tau_eta ))
+  The same formula applies for q_gamma.
+
+  On the postsynapic side, there can be arbitrarily many synaptic time constants
+  (gif_psc_exp has exactly two: tau_syn_ex and tau_syn_in). This can be reached
+  by specifying separate receptor ports, each for a different time constant. The
+  port number has to match the respective "receptor_type" in the connectors.
+
+  The shape of synaptic conductance is exponential.
+
+  Parameters:
+  The following parameters can be set in the status dictionary.
+
+  Membrane Parameters:
+    C_m        double - Capacity of the membrane in pF
+    t_ref      double - Duration of refractory period in ms.
+    V_reset    double - Reset value after a spike in mV.
+    E_L        double - Leak reversal potential in mV.
+    g_L        double - Leak conductance in nS.
+    I_e        double - Constant external input current in pA.
+
+  Spike adaptation and firing intensity parameters:
+    q_stc      vector of double - Values added to spike-triggered currents (stc)
+                                  after each spike emission in nA.
+    tau_stc    vector of double - Time constants of stc variables in ms.
+    q_sfa      vector of double - Values added to spike-frequency adaptation
+                                  (sfa) after each spike emission in mV.
+    tau_sfa    vector of double - Time constants of sfa variables in ms.
+    Delta_V    double - Stochasticity level in mV.
+    lambda_0   double - Stochastic intensity at firing threshold V_T in 1/s.
+    V_T_star   double - Minimum threshold in mV
+
+  Synaptic parameters
+    taus_syn   vector of double - Time constants of the synaptic conductance
+                                  in ms.
+    E_ex       double - Excitatory reversal potential in mV.
+    E_in       double - Inhibitory reversal potential in mV.
+
+  References:
+
+  [1] Mensi S, Naud R, Pozzorini C, Avermann M, Petersen CC, Gerstner W (2012)
+  Parameter extraction and classification of three cortical neuron types
+  reveals two distinct adaptation mechanisms. J. Neurophysiol., 107(6),
+  1756-1775.
+
+  [2] Pozzorini C, Mensi S, Hagens O, Naud R, Koch C, Gerstner W (2015)
+  Automated High-Throughput Characterization of Single Neurons by Means of
+  Simplified Spiking Models. PLoS Comput. Biol., 11(6), e1004275.
+
+  Sends: SpikeEvent
+
+  Receives: SpikeEvent, CurrentEvent, DataLoggingRequest
+
+  Author: March 2016, Setareh
+  SeeAlso: pp_psc_delta, gif_cond_exp, gif_cond_exp, gif_cond_exp_multisynapse
```

Comment:
``gif_cond_exp`` is referenced twice + self reference.
Maybe
```c++
SeeAlso: pp_psc_delta, gif_cond_exp, iaf_psc_exp_multisynapse, gif_psc_exp_multisynapse
```
would be more appropriate.